### DISCUSSION TITLE
are generics fully supported?

### DISCUSSION THREAD

**Discussion by Salmaan21** [golangci/golangci-lint#3580] [Q&A] [answered]
I was looking at the support of the generics issue https://github.com/golangci/golangci-lint/issues/2649 and saw that the last linter (sqlclosecheck) recently updated the code to support generics so, is it fully supported now with the new release? Because it still shows 7/13 even though there seems to have been a lot more progress.

**Comment by Antonboom** [comment, marked as answer]
@Salmaan21, hello!

I updated the list in the issue.
Looks like that `rowserrcheck` is the last of linters needed [to be supported](https://github.com/jingyugao/rowserrcheck/issues/24#issuecomment-1200964592).

I'm waiting for https://github.com/jingyugao/rowserrcheck/pull/28

**Comment by Antonboom** [comment]
https://github.com/golangci/golangci-lint/issues/2649 closed 🎉 🎉
